src/components/[guild]/activity/ActivityLogContext.tsx

```diff
@@ -123,9 +148,25 @@ const ActivityLogProvider = ({
       })
     }
 
+    if (!query.action) searchParams = addActionGroupFilterParams(searchParams)
+
     return `/auditLog?${searchParams.toString()}`
   }
 
+  const addActionGroupFilterParams = (searchParams: URLSearchParams) => {
```

Comment:
This function just appends search params to `searchParams`, so I think we don't need to return this variable from it, that way we could use `const searchParams` in line 135 (instead of `let`)